tests/lib/services/youtube.test.js

```diff
@@ -27,14 +35,37 @@ describe('Youtube Tests', () => {
                     data: mockResponses.getListOfUploadedVideos
                 });
             }
+        },
+        search: {
+            list: function(){
+                searchCalled += 1;
+
+                if (searchCalled % 2 === 1) {
+                    return Promise.resolve({ data: mockResponses.getActiveLiveBroadcastsVideoId });
+                }
+                if (searchCalled % 2 === 0) {
```

Comment:
This should be changed to something that isn't extremely fragile, but it works for now